kuma/javascript/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+// @flow
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Logo from './Logo';
+
+const container = document.getElementById('main-header-logo');
+
+if (container) {
+    ReactDOM.render(<Logo url={container.dataset.url}/>, container);
```

Comment:
With SSR, shouldn't this become a hydrate() call instead of a render() call? Or am I missing something?

Reply:
@davidflanagan This part isn't touched at all on SSR side, it's for frontend only. This module isn't even imported in SSR.